lib/Onnxifi/onnxifiGlow.cpp

```diff
@@ -59,6 +59,11 @@ GLOW_ONNXIFI_LIBRARY_FUNCTION_WRAPPER(onnxGetBackendIDs)(
 #else
   constexpr bool withCPU = false;
 #endif
+#ifdef GLOW_WITH_HABANA
```

Comment:
I think changes in this file are good to go, but we need to have a principled way of handling non open sourced backend in the onnxifi glow.